src/cards/vacuum-card/vacuum-card-config.ts

```diff
@@ -31,6 +32,7 @@ export const vacuumCardConfigStruct = assign(
     lovelaceCardConfigStruct,
     assign(entitySharedConfigStruct, appearanceSharedConfigStruct, actionsSharedConfigStruct),
     object({
+        display_battery: optional(boolean()),
```

Comment:
```suggestion
        show_battery_icon: optional(boolean()),
```
Because we already have `show_conditons`, `show_temperature` for weather chip for example.